### DISCUSSION TITLE
Release 0.6.5 "Testimony": Small bug fixes

### DISCUSSION THREAD

**Discussion by c3d** [c3d/db48x#819] [Announcements]
This release does not contain much because FOSDEM took a lot of energy.

## New features

* menu: Connect `ceil` and `floor` functions
* Add real to polar conversions
* units: Add `dms` unit to angles menu

## Bug fixes

* decimal: Compute `ln(0.002)` correctly
* integer: Do not parse degree sign if in a complex
* units: Fix parsing of angle units
* font: Add radian glyph

**Full Changelog**: https://github.com/c3d/DB48X-on-DM42/compare/v0.6.4...v0.6.5

<hr /><em>This discussion was created from the release <a href='https://github.com/c3d/DB48X-on-DM42/releases/tag/v0.6.5'>Release 0.6.5 "Testimony": Small bug fixes</a>.</em>

**Comment by evrimoztamur** [comment]
Your talk was one of my favourites at FOSDEM, thank you, and godspeed!
